exoctk/exoctk_app/form_validation.py

```diff
@@ -46,7 +46,7 @@ class FortneyModelForm(BaseForm):
 class LimbDarkeningForm(BaseForm):
     """Form validation for the limb_darkening tool"""
     # Model grid
-    default_modelgrid = os.path.join(MODELGRID_DIR, 'ATLAS9/')
+    default_modelgrid = os.path.join(MODELGRID_DIR)
```

Comment:
@jaymedina was the `ATLAS9/` directory wrapped up into the `default` directory when the environment variables were refactored?

Reply:
Yep! That's why the website wouldn't launch -- there was no `ATLAS9` sub-directory.

Reply:
Cool -- just wanted to verify!